website/docs/cloud-docs/api-docs/invoices.mdx

```diff
@@ -1,7 +1,8 @@
 ---
-page_title: Invoices - API Docs - HCP Terraform
+page_title: Invoices API for HCP Terraform
 description: >-
-  Use the `invoices` endpoint to access an organization's invoices. List previous invoices and get the next invoice using the HTTP API.
+  Use HCP Terraform API's `invoices` endpoint to read an organization's current, past, and future invoices.
+tfc_only: true
```

Comment:
I know this is cheating a bit -- but the very first note is that this is only for HCP TF - so it feels like it'd be easier to just exclude it